classes/Segmentation.php

```diff
@@ -891,20 +896,20 @@ public function getQuery($post, $live, $limit = false, $speField = '')
 			}
 		}
 
+
+			if($post['sourceSelect'][0] == 4){
+				$multistoreWhere = ' AND c.id_shop = ' . $post['fieldSelect'][0];
```

Comment:
Can you please sanitize the data inserted in the SQL request with an `(int)` cast ?